SpatialGDK/Source/SpatialGDKFunctionalTests/Private/GenerateTestMapsCommandlet.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "GenerateTestMapsCommandlet.h"
+#include "TestMaps/GeneratedTestMap.h"
+
+DEFINE_LOG_CATEGORY(LogGenerateTestMapsCommandlet);
+
+UGenerateTestMapsCommandlet::UGenerateTestMapsCommandlet()
+{
+	IsClient = false;
+	IsEditor = true;
+	IsServer = false;
+	LogToConsole = true;
+}
+
+int32 UGenerateTestMapsCommandlet::Main(const FString& CmdLineParams)
+{
+	UE_LOG(LogGenerateTestMapsCommandlet, Display, TEXT("Generate test maps commandlet started."));
+
+	UE_LOG(LogGenerateTestMapsCommandlet, Display, TEXT("Deleting the %s folder."), *UGeneratedTestMap::GetGeneratedMapFolder());
+	if (FPaths::DirectoryExists(UGeneratedTestMap::GetGeneratedMapFolder()))
+	{
+		IPlatformFile& PlatformFile = FPlatformFileManager::Get().GetPlatformFile();
+		const bool bSuccess = PlatformFile.DeleteDirectoryRecursively(*UGeneratedTestMap::GetGeneratedMapFolder());
```

Comment:
I wonder how this will play with p4 locking files, defo should be in untracked folder locations to avoid this